docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,202 @@
+
+* undocumented fields in object with schemas are silently dropped
+* objects without schemas are serialized into a json string
+* `_airbyte_additional_properties` is dropped entirely
+
+For example, the following input schemas and data will result in the following outputs:
+
+| Input Schema                                                          | Input Data                     | Output Schema                                                         | Output Data                          |
+|-----------------------------------------------------------------------|--------------------------------|-----------------------------------------------------------------------|--------------------------------------|
+| `{ "type": "object", "properties": { "id": { "type": "integer" } } }` | `{ "id": 1, "name": "Alice" }` | `{ "type": "object", "properties": { "id": { "type": "integer" } } }` | `{ "id": 1 }`                        |
+| `{ "type": "object" }`                                                | `{ "id": 1, "name": "Alice" }` | `{ "type": "string" }`                                                | `"{\"id\": 1, \"name\": \"Alice\"}"` |
+
+This behavior will be applied to both Avro and Parquet formats.
+
+Note: dropped fields will not appear in `_airbyte_meta.changes[]`.
+
+### Array types
+
+Formerly, arrays without types (json-schema with `"type": "array"` but no `items` field) were converted to native arrays of string serializations of the underlying types. The same behavior was applied to arrays of union types (`{ "type": "array", "items": { "oneOf": [ /* various types */ ] }`).
+
+Now:
+
+* arrays without types are serialized into json array strings
+* arrays of unions are treated as arrays of mixed types
+
+For example, the following input schemas and data formerly resulted in:
+
+| Input Schema                                                                           | Input Data     | Old Output Schema                                    | New Output Data  |
+|----------------------------------------------------------------------------------------|----------------|------------------------------------------------------|------------------|
+| `{ "type": "array", "items": { "type": "integer" } }`                                  | `[1, "Alice"]` | `{ "type": "array", "items": ["null", "integer"] }`  | [SYNC FAILED]    |
+| `{ "type": "array" }`                                                                  | `[1, "Alice"]` | `{ "type": "array", "items": [ "null", "string" ] }` | `["1", "Alice"]` |
+| `{ "type": "array", "items": { "oneOf": [ {"type": "integer", "type": "string"} ] } }` | `[1, "Alice"]` | `{ "type": "array", "items": [ "null", "string" ] }` | `["1", "Alice"]` |
+| `{ "type": "array", "items": { "oneOf": [ {"type": "integer", "type": "string"} ] } }` | `[1, false]`   | `{ "type": "array", "items": [ "null", "string" ] }` | `["1", "false"]` |
+
+Now:
```

Comment:
is this intentionally not combined with the previous table? (I'm guessing it's just too wide for comfortable reading?)

Reply:
Yes just too wide